src/types/QIX.ts

```diff
@@ -40,8 +40,8 @@ export interface RowHeight {
 }
 
 export interface PaletteColor {
-  index: number;
-  color: string | null;
+  index?: number | undefined;
```

Comment:
No need for `| undefined` when the property is optional

Reply:
I'm using the same typing from the stardust api, but It works fine without ` | undefined` so I will remove it